roles/openldap/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- include: install-configure.yml
+- include: password_policy.yml
+- include: setup_check_password.yml
+  when: openldap.password_policy_enabled|bool
+- include: setup_sync_provider.yml
```

Comment:
The only reason these should be split out is if different tagging will be applied to the tasks, or a conditional will be applied, or if they are set aside across all roles (like logging). Otherwise all the things should be in the main.yml task file for easy discovery.